akka-stream/src/main/scala/akka/stream/impl/fusing/Fusing.scala

```diff
@@ -560,7 +560,7 @@ private[stream] object Fusing {
      * connections within imported (and not dissolved) GraphModules.
      * See also the comment in addModule where this is partially undone.
      */
-    def registerInteral(s: Shape, indent: Int): Unit = {
+    def registerInternal(s: Shape, indent: Int): Unit = {
       if (Debug) println("  " * indent + s"registerInternals(${s.outlets.map(hash)})")
```

Comment:
Oh wait, by looking at the debug println I think this method should actually be called `registerInternals` - could you fix that up in this PR @jtescher please?